mozilla/doh-rollout
# Fixed #152  – Sets local storage pref to remember that user has opted out of study manually

### To verify: 

- Install add-on and turn DoH on per normal directions
- Click "OK" when presented with door hanger
- Confirm `network.trr.mode` is set to `2`
- Disable DoH from the about:preferences Network dialog box 
- Confirm `network.trr.mode` is set to `0`
- Restart browser. Remember to run Normandy commands each time on start-up. 
- Confirm `network.trr.mode` is set to `0` after 2+ restarts
